tools/Az.Tools.Predictor/Az.Tools.Predictor.Test/AzPredictorTests.cs

```diff
@@ -37,12 +37,12 @@ public sealed class AzPredictorTests
         /// </summary>
         public AzPredictorTests(ModelFixture modelFixture)
         {
-            this._fixture = modelFixture;
-            var startHistory = $"{AzPredictorConstants.CommandPlaceholder}{AzPredictorConstants.CommandConcatenator}{AzPredictorConstants.CommandPlaceholder}";
+            _fixture = modelFixture;
+            var starory = $"{AzPredictorConstants.CommandPlaceholder}{AzPredictorConstants.CommandConcatenator}{AzPredictorConstants.CommandPlaceholder}";
```

Comment:
It seems this variable name was accidentally updated? If not, I am unclear what is a starory.